### ISSUE TITLE
New OutputMap function for the Terraform module

### ISSUE THREAD

**Issue by jmfontaine** [gruntwork-io/terratest#175]
I am considering adding a set of functions to return the Terraform output for a map, similar to the recently added [OutputList and OutputListE functions](https://github.com/gruntwork-io/terratest/pull/169).

That was actually [suggested in the PR](https://github.com/gruntwork-io/terratest/pull/169#discussion_r224589733) and I have a need for it: I have a Terraform module that generates standardized AWS tags and outputs them as a map.

The function signatures would be:

```go
OutputMap(t *testing.T, options *Options, key string) [string]string
OutputMapE(t *testing.T, options *Options, key string) ([string]string, error)
```

If you think it would be valuable to the community, I can submit a PR for that feature.

**Comment by brikis98** [comment]
That would indeed be very handy. PR welcome. Thank you!

**Comment by jmfontaine** [comment]
Will submit a PR soon then. Thanks for the feedback!

**Comment by jmfontaine** [comment]
This was fixed by PR #180. Closing.
